scripts/autobahn-server.sh

```diff
@@ -14,18 +13,22 @@ trap cleanup TERM EXIT
 
 function test_diff() {
     if ! diff -q \
-        <(jq -S 'del(."Tungstenite" | .. | .duration?)' 'autobahn/client-results.json') \
+        <(jq -S 'del(."Tungstenite" | .. | .duration?)' 'autobahn/expected-results.json') \
         <(jq -S 'del(."Tungstenite" | .. | .duration?)' 'autobahn/server/index.json')
     then
-        echo Difference in results, either this is a regression or \
-             one should update autobahn/server-results.json with the new results.
+        echo 'Difference in results, either this is a regression or' \
+             'one should update autobahn/expected-results.json with the new results.' \
         exit 64
     fi
 }
 
-cargo build --release --example autobahn-server
 cargo run --release --example autobahn-server & WSSERVER_PID=$!
-echo "Server PID: ${WSSERVER_PID}"
 sleep 3
-wstest -m fuzzingclient -s 'autobahn/fuzzingclient.json'
+
+docker run -d --rm \
```

Comment:
> > Applies also to scripts/autobahn-server.sh.
> Does it? I'm asking because in case of autobahn-server (...) we use Autobahn only for the fuzzing client. Once the client process ends, the container is stopped.

Oh I see, you're right.

In this case, we should _not_ add `-d` to `docker run` flags here. The `-d` causes that `docker run` command is non-blocking, so `test_diff` below would be run too early in that case IMO. (I wonder how that could work in current PR state).